js/main.js

```diff
@@ -0,0 +1,141 @@
+"use strict";
+
+const TITLES = [
+  `Дворец из коробки`,
+  `Квартира без окон`,
+  `Квартирка с видом на автобусную остановку`,
+  `Бунгало у фонтана`,
+  `Дом трехэтажный с бассейном`,
+  `Просторный гараж`
+];
+const HOUSES_TYPES = [
```

Comment:
Так вернее
```suggestion
const HOUSE_TYPES = [
```
Также `PHOTO_URLS`, но `PINS_AMOUNT`